community/gnupg2/sources

```diff
@@ -0,0 +1 @@
+https://github.com/gpg/gnupg/archive/gnupg-2.2.19.tar.gz
```

Comment:
This source should allow you to remove the need for `autoreconf`. 
```
https://gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.19.tar.bz2
```

Reply:
done